pilz_control/test/pjtc_test_helper.h

```diff
@@ -155,6 +155,33 @@ static bool updateUntilRobotMotion(RobotDriver* robot_driver,
                  [robot_driver]() { robot_driver->update(); });
 }
 
+//! @brief Checks for the specified observation time that the robot does not move.
```

Comment:
what's "observation time"? Some seconds in the past? Or an absolute timestamp in the future?

Reply:
I agree on this one. The interface could be misunderstood 
since observation_time is passed as `std::chrono::milliseconds` but it is used as a duration.
I suggest using `const std::chrono::milliseconds`

Reply:
But well since std::chrono::milliseconds is just a typedef for some std::chrono::duration... nevermind. But I updated the parameter to state the intent.

Reply:
Resolved since outdated.